pkgs/by-name/vz/vzic/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  glib,
+  tzdata,
+  libical,
+}:
+stdenv.mkDerivation rec {
+  pname = "vzic";
+  version = "2024a";
+
+  src = fetchFromGitHub {
+    owner = "libical";
+    repo = "vzic";
+    rev = "4a87e2bed237585cbcf209507d0b82c023113633";
+    hash = "sha256-zRy62SJa8XUBPnoW/fgANKZaatNWqmXka4LgRUlbLzw=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    tzdata
+    glib
+    libical
+  ];
+  env = {
+    OLSON_DIR = "tzdata" + version;
```

See note about `version`.

```suggestion
    OLSON_DIR = "tzdata2024a";
```